observability/logging/cluster-logging-support.adoc

```diff
@@ -34,19 +34,31 @@ LokiStack development is ongoing. Not all APIs are currently supported.
 
 |LokiStack
 |lokistack.loki.grafana.com/v1
-|Supported in 5.5
+|Supported from 5.5
 
 |RulerConfig
 |rulerconfig.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
 
 |AlertingRule
 |alertingrule.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
 
 |RecordingRule
 |recordingrule.loki.grafana/v1
-|Supported in 5.7
+|Supported from 5.7
+
+|LogFileMetricExporter
+|LogFileMetricExporter.logging.openshift.io/v1alpha1
+|Supported from 5.8
+
+|ClusterLogForwarder
+|clusterlogforwarder.observability.openshift.io/v1
```

Comment:
```suggestion
|clusterlogforwarder.logging.openshift.io/v1
```

Reply:
or seeing the following section.. depending if this is supposed to be 6.x centric.... it can be removed.... if not 6.x centric it should say supported since 4.5 and be 'clusterlogforwarder.logging.openshift.io/v1'

Reply:
We want to keep this section common to 5.8 and 6.y releases. So how about:

clusterlogforwarder.logging.openshift.io/v1
Supported from 4.5. Removed in 6.0.